assignment1/Assignment1.ipynb

```diff
@@ -1111,7 +1111,16 @@
       "source": [
         "# Please note that this code needs only to be run in a fresh runtime.\n",
         "# However, it can be rerun afterwards too.\n",
+        "# Here we download and install localy two libraries:\n",
+        "# -  gdown is library similar to curl and wget but allow to download big files from google drive\n",
+        "#    two previous apps have problem with this because of google warning that file is too big to scan it with antyvirus.\n",
+        "# -  httpimport allow to import modules from web using HTTP/HTTPS.\n",
         "!pip install -q gdown httpimport\n",
+        "# If you don't use virtual env and you work on own computer, then you can want to install this packages\n",
+        "# system wide, in such case you should use: 'sudo pip install gdown httpimport' in your shell.\n",
+        "# Download MNIST data set, which we will be use in problem 5..\n",
+        "# If you have installed gdown localy, then you want probably to change here 'gdown' to '~/.local/bin/gdown',\n",
+        "# because ~/.local/bin is default directory for packages installed localy by pip.\n",
```

Comment:
```suggestion
        "# If you don't use virtual env and you work on own computer, then you may want to install these packages\n",
        "# system wide, in such case you should use: 'sudo pip install gdown httpimport' in your shell.\n",
        "# Download MNIST data set, which we will be use in problem 5..\n",
        "# If you have installed gdown locally, then you probably want to change here 'gdown' to '~/.local/bin/gdown',\n",
        "# because ~/.local/bin is default directory for packages installed locally by pip.\n",
```